docs/customization/diff.md

```diff
@@ -0,0 +1,55 @@
+# Diffing Customization
+
+It is possible for an application to still be `OutOfSync` even immediately after a successful Sync
+operation. Some reasons for this might be:
+* There is a bug in the manifest itself, where it contains extra/unknown fields from the 
+  actual K8s spec. These extra fields would get dropped when querying Kubernetes for the live state,
+  resulting in an `OutOfSync` status indicating a missing field was detected.
+* The sync was performed (with pruning disabled), and there are resources which need to be deleted.
+* A controller or [mutating webhook](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#mutatingadmissionwebhook)
+  is altering the object after it was submitted to Kubernetes in a manner which contradicts git.
+* A helm chart is using a template function such as [`randAlphaNum`](https://github.com/helm/charts/blob/master/stable/redis/templates/secret.yaml#L16),
+  which generates different data every time `helm template` is invoked.
+
+In case if it is impossible to fix upstream issue Argo CD allows to optionally ignore differences of problematic resources. The diffing customization can be configured
+for a single or multiple application resources or at a system level.
+
+## Application level configuration
+
+Argo CD allows ignoring differences at a specific json path. Following sample application is configured to ignore difference if `spec.replicas` for all deployments.
+
+```yaml
+spec:
+  ignoreDifferences:
+  - group: apps
+    kind: Deployment
+    json6902Paths:
```

Comment:
The terminology `json6902` is actually incorrect and refers to a style of patching. I think we can refer to this as simply jsonPointer. See: https://tools.ietf.org/html/rfc6901

Reply:
Glad you found it. Renamed to jsonPointer in documentation as well as in implementation PR https://github.com/argoproj/argo-cd/pull/1130